### PR TITLE
move to json

### PR DESCRIPTION
**Pull request by alinelena**
move to the new info.json approach

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mlego/m65/rev1/info.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "unicodemap": true
+    },
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:18]:
```suggestion
```

File: keyboards/mlego/m65/rev1/info.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    "matrix_size": {
+        "cols": 13,
+        "rows": 5
+    },
+    "mouse_key": {
+        "enabled": true
+    },
+    "parse_errors": [],
+    "parse_warnings": [],
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:53]:
```suggestion
```

File: keyboards/mlego/m65/rev1/info.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    "processor": "STM32F103",
+    "processor_type": "arm",
+    "protocol": "ChibiOS",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:56]:
```suggestion
```

File: keyboards/mlego/m65/rev1/info.json [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        "led_count": 20,
+        "pin": "B15",
+        "saturation_steps": 8
+    },
+    "tapping": {
+        "toggle": 2
+    },
+    "url": "https://gitlab.com/m-lego/m65",
+    "usb": {
+        "device_ver": "0x0001",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:83]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Same should apply to all the other revisions.

File: keyboards/mlego/m48/rev1/info.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m48/rev1",
```
Comment (zvecr) [inline keyboards/mlego/m48/rev1/info.json:25]:
```suggestion
```

File: keyboards/mlego/m48/rev1/info.json [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    "matrix_pins": {
+        "cols": ["A10", "A15", "B3", "B4", "B5", "B7", "B6", "A1", "A2", "A3", "A4", "A5"],
+        "rows": ["A6", "A7", "B0", "B10"]
+    },
+    "matrix_size": {
+        "cols": 12,
+        "rows": 4
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m48/rev1/info.json:34]:
```suggestion
```

File: keyboards/mlego/m60/rev1/info.json [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m60/rev1",
```
Comment (zvecr) [inline keyboards/mlego/m60/rev1/info.json:25]:
```suggestion
```

File: keyboards/mlego/m60/rev1/info.json [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+    "matrix_pins": {
+        "cols": ["A10", "A15", "B3", "B4", "B5", "B7", "B6", "A1", "A2", "A3", "A4", "A5"],
+        "rows": ["A6", "A7", "B0", "B1", "B10"]
+    },
+    "matrix_size": {
+        "cols": 12,
+        "rows": 5
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m60/rev1/info.json:34]:
```suggestion
```

File: keyboards/mlego/m60_split/rev1/info.json [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m60_split/rev1",
```
Comment (zvecr) [inline keyboards/mlego/m60_split/rev1/info.json:25]:
```suggestion
```

File: keyboards/mlego/m60_split/rev1/info.json [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+    "matrix_pins": {
+        "cols": ["B14", "A10", "A15", "B3", "B4", "B5"],
+        "rows": ["B0", "A6", "A7", "B1", "A5"]
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m60_split/rev1/info.json:30]:
```suggestion
```

File: keyboards/mlego/m60_split/rev2/info.json [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m60_split/rev2",
```
Comment (zvecr) [inline keyboards/mlego/m60_split/rev2/info.json:25]:
```suggestion
```

File: keyboards/mlego/m60_split/rev2/info.json [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+    "matrix_pins": {
+        "cols": ["B14", "A10", "A15", "B3", "B4", "B5"],
+        "rows": ["B0", "A6", "A7", "B1", "A5"]
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m60_split/rev2/info.json:30]:
```suggestion
```

File: keyboards/mlego/m65/rev1/info.json [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m65/rev1",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:25]:
```suggestion
```

File: keyboards/mlego/m65/rev1/info.json [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+    "matrix_pins": {
+        "cols": ["A10", "A15", "B3", "B4", "B5", "B9", "B8", "B7", "B6", "C15", "A0", "A7", "B10"],
+        "rows": ["B11", "B0", "B1", "A2", "A3"]
+    },
+    "matrix_size": {
+        "cols": 13,
+        "rows": 5
+    },
```
Comment (zvecr) [inline keyboards/mlego/m65/rev1/info.json:33]:
```suggestion
```

File: keyboards/mlego/m65/rev2/info.json [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "B2",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m65/rev2",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev2/info.json:25]:
```suggestion
```

File: keyboards/mlego/m65/rev2/info.json [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+    "matrix_pins": {
+        "cols": ["A10", "A15", "B3", "B4", "B5", "B9", "B8", "B7", "B6", "C15", "A0", "A7", "B10"],
+        "rows": ["B11", "B0", "B1", "A2", "A3"]
+    },
+    "matrix_size": {
+        "cols": 13,
+        "rows": 5
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev2/info.json:34]:
```suggestion
```

File: keyboards/mlego/m65/rev3/info.json [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m65/rev3",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev3/info.json:25]:
```suggestion
```

File: keyboards/mlego/m65/rev3/info.json [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+    "matrix_pins": {
+        "cols": ["A10", "A15", "B3", "B4", "B5", "B9", "B8", "B7", "A1", "A2", "A3", "A4", "B1"],
+        "rows": ["B10", "A5", "A6", "A7", "B0"]
+    },
+    "matrix_size": {
+        "cols": 13,
+        "rows": 5
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev3/info.json:34]:
```suggestion
```

File: keyboards/mlego/m65/rev4/info.json [hunk 19]
```diff
@@ -0,0 +1,139 @@
...
+        "oled": true,
+        "rgblight": true,
+        "wpm": true
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "num_lock": "B12",
+        "scroll_lock": "B13"
+    },
+    "keyboard_folder": "mlego/m65/rev4",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev4/info.json:27]:
```suggestion
```

File: keyboards/mlego/m65/rev4/info.json [hunk 20]
```diff
@@ -0,0 +1,139 @@
...
+    "matrix_pins": {
+        "cols": ["B14", "A8", "A10", "A15", "B3", "B4", "B5", "B7", "A1", "A2", "A3", "A4", "B1"],
+        "rows": ["B10", "A5", "A6", "A7", "B0"]
+    },
+    "matrix_size": {
+        "cols": 13,
+        "rows": 5
+    },
+    "platform": "STM32",
```
Comment (zvecr) [inline keyboards/mlego/m65/rev4/info.json:36]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by alinelena** [comment]
> Same should apply to all the other revisions.

done

**Pull request closed without merging**